[2025-02-04T07:34:59.555-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-04T07:34:59.577-0500] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_reporte_como_vamos.bot_ex_bbva manual__2025-02-04T12:17:16.570936+00:00 [queued]>
[2025-02-04T07:34:59.588-0500] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_reporte_como_vamos.bot_ex_bbva manual__2025-02-04T12:17:16.570936+00:00 [queued]>
[2025-02-04T07:34:59.590-0500] {taskinstance.py:2866} INFO - Starting attempt 4 of 4
[2025-02-04T07:34:59.607-0500] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): bot_ex_bbva> on 2025-02-04 12:17:16.570936+00:00
[2025-02-04T07:34:59.614-0500] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=26458) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-04T07:34:59.616-0500] {standard_task_runner.py:72} INFO - Started process 26459 to run task
[2025-02-04T07:34:59.617-0500] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_reporte_como_vamos', 'bot_ex_bbva', 'manual__2025-02-04T12:17:16.570936+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/etl_como_vamos_dag.py', '--cfg-path', '/tmp/tmpol5h62jv']
[2025-02-04T07:34:59.619-0500] {standard_task_runner.py:105} INFO - Job 42: Subtask bot_ex_bbva
[2025-02-04T07:34:59.674-0500] {task_command.py:467} INFO - Running <TaskInstance: etl_reporte_como_vamos.bot_ex_bbva manual__2025-02-04T12:17:16.570936+00:00 [running]> on host 21f1c1b48730
[2025-02-04T07:34:59.769-0500] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_reporte_como_vamos' AIRFLOW_CTX_TASK_ID='bot_ex_bbva' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T12:17:16.570936+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-04T12:17:16.570936+00:00'
[2025-02-04T07:34:59.770-0500] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-04T07:34:59.795-0500] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-04T07:34:59.797-0500] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/scripts/bot_etl_ex_bbva.py']
[2025-02-04T07:34:59.810-0500] {subprocess.py:99} INFO - Output:
[2025-02-04T07:35:00.807-0500] {subprocess.py:106} INFO - 2025-02-04 07:35:00,807 - bot_etl_ex_bbva - INFO - Bot ETL Exito-BBVA started
[2025-02-04T07:35:01.715-0500] {subprocess.py:106} INFO - 2025-02-04 07:35:01,715 - bot_etl_ex_bbva - INFO - Conexi√≥n al Selenium Hub establecida correctamente
[2025-02-04T07:35:02.667-0500] {subprocess.py:106} INFO - 2025-02-04 07:35:02,667 - bot_etl_ex_bbva - INFO - Credentials input success
[2025-02-04T07:35:03.772-0500] {subprocess.py:106} INFO - 2025-02-04 07:35:03,772 - bot_etl_ex_bbva - INFO - Click in login button
[2025-02-04T07:35:13.796-0500] {subprocess.py:106} INFO - 2025-02-04 07:35:13,796 - bot_etl_ex_bbva - INFO - Loging success
[2025-02-04T07:35:14.784-0500] {subprocess.py:106} INFO - 2025-02-04 07:35:14,784 - bot_etl_ex_bbva - INFO - Input file succes
[2025-02-04T07:35:24.888-0500] {subprocess.py:106} INFO - 2025-02-04 07:35:24,886 - bot_etl_ex_bbva - INFO - Click in Cargar
[2025-02-04T07:35:30.626-0500] {subprocess.py:106} INFO - 2025-02-04 07:35:30,625 - bot_etl_ex_bbva - INFO - bot_etl_ex_bbva finish execution
[2025-02-04T07:35:30.812-0500] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-04T07:35:30.870-0500] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-04T07:35:30.872-0500] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_reporte_como_vamos, task_id=bot_ex_bbva, run_id=manual__2025-02-04T12:17:16.570936+00:00, execution_date=20250204T121716, start_date=20250204T123459, end_date=20250204T123530
[2025-02-04T07:35:30.941-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-04T07:35:30.985-0500] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-04T07:35:30.989-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
